shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart-jit-runner-integration-test.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The generated C++ bindings for the Echo FIDL protocol
+#include <flutter/shell/platform/fuchsia/dart_runner/tests/fidl/flutter.example.echo/flutter/example/echo/cpp/fidl.h>
+
+#include <lib/async-loop/testing/cpp/real_loop.h>
+#include <lib/sys/component/cpp/testing/realm_builder.h>
+#include <lib/sys/component/cpp/testing/realm_builder_types.h>
+
+#include "flutter/fml/logging.h"
+#include "gtest/gtest.h"
+
+using namespace component_testing;
```

Comment:
This is discouraged by Google's C++ style guide: https://google.github.io/styleguide/cppguide.html#Namespaces

Prefer being explicit on what you're using:

```
using component_testing::Foo;
using component_testing::Bar;
```